app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:id])
+    @comment = Comment.new
+  end
+
```

Comment:
Great job @MarcoOquendoC 🤝

- [ ] [Optional] Kindly add a params method to help prevent accidentally allowing users to update sensitive model attributes. You can learn more about strong parameters by following this [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides).
![params-comment](https://user-images.githubusercontent.com/63917725/215305978-24fc8c79-00e2-45a7-8143-c56b824d3d16.PNG)